astro_2.0/components/BountyCard/tests/BountyCard.spec.tsx

```diff
@@ -10,6 +10,60 @@ import { BountyCard } from 'astro_2.0/components/BountyCard';
 import { BountyCardContent } from 'astro_2.0/components/BountyCard/types';
 
 describe('bounty card', () => {
+  const dao = {
```

Comment:
There is a mock file near by the test. I would suggest to place this data there to reduce size of the test.